content/en/blog/_posts/2021-04-14-local-storage-features-go-beta.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: blog
+title: "Local Storage: Storage Capacity Tracking, Distributed Provisioning and Generic Ephemeral Volumes hit Beta"
+date: 2021-04-14
+slug: local-storage-features-go-beta
+---
+
+ **Authors:** Patrick Ohly (Intel)
+
+The ["generic ephemeral
+volumes"](/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes)
+and ["storage capacity
+tracking"](/docs/concepts/storage/storage-capacity/)
+features in Kubernetes are getting promoted to beta in Kubernetes
+1.21. Together with the [distributed provisioning
+support](https://github.com/kubernetes-csi/external-provisioner#deployment-on-each-node)
+in the CSI external-provisioner, development and deployment of
+Container Storage Interface (CSI) drivers which manage storage locally
+on a node become a lot easier.
+
+This blog post explains how such drivers worked before and how these
+features can be used to make drivers simpler.
+
+## Current status
```

Comment:
I would rename this section to "Problems we're solving"

Reply:
Actually reading through some more, this section is quite long and talks about very different issues (with different solutions). Would it flow better to treat each issue as its own section and talk about problem and solution together in each section?

Reply:
I raised that question earlier myself (https://github.com/kubernetes/website/pull/27240#issuecomment-808245490, "Readers have to remember...") but got no response. I'm unsure whether a different structure would work better, therefore I prefer to keep it like this.

Reply:
Let's see if it becomes more readable with the new section title and subsections...